src/aks-preview/azcli-aks-live-test/build_image.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -eux
+pwd
+
+# build docker image
+docker build -t azcli-aks-live-test-image:latest -f ./Dockerfile .
```

Comment:
remove this after we have our own image.

Reply:
A new variable is added to control the options for building the image.  If there is no pre-built image or the pull fails, it will fall back to build locally.